carbon-design-system/carbon
# fix: ailabel autoalign was not working

Closes #17571 

AILable auto align was not working

#### Changelog

**New**

Added `shift` in middleware of popover component. It prevents the floating element from overflowing along its axis of alignment, thereby preserving the side it’s placed on.


#### Testing / Reviewing

1. Go to storybook > AILable > Default Story
2. Click on AI Label, AILabelContent should become fully visible and avoid overflowing along its axis..
